cpp/src/arrow/compute/kernels/vector_selection_benchmark.cc

```diff
@@ -322,6 +367,36 @@ static void TakeFixedSizeBinaryMonotonicIndices(benchmark::State& state) {
       .FixedSizeBinary();
 }
 
+static void TakeChunkedChunkedInt64RandomIndicesNoNulls(benchmark::State& state) {
+  TakeBenchmark(state, false)
```

Comment:
Can we state bool arg names consistently? I mean `/*indices_with_nulls=*/`

Reply:
I will add to this and all the other cases.